experiment/nursery/create_bazel_cache_rcs.sh

```diff
@@ -37,26 +38,35 @@ command_to_version () {
     package_to_version $PACKAGE
 }
 
+hash_toolchains () {
+    # if $CC is set bazel will use this to detect c/c++ toolchains, otherwise gcc
+    # https://blog.bazel.build/2016/03/31/autoconfiguration.html
+    CC="${CC:-gcc}"
+    CC_VERSION=$(command_to_version $CC)
+    # NOTE: IIRC some rules call python internally, this can't hurt
+    PYTHON_VERSION=$(command_to_version python)
+    # combine all tool versions into a hash
+    # NOTE(bentheelder): if we change the set of tools considered we should
+    # consider prepending the hash with a """schema version""" for completeness
+    TOOL_VERSIONS="CC:${CC_VERSION},PY:{PYTHON_VERSION}"
+    echo "${TOOL_VERSIONS}" | md5sum | cut -d" " -f1
+}
+
 make_bazel_rc () {
     # this is the default for recent releases but we set it explicitly
     # since this is the only hash our cache supports
     echo "startup --host_jvm_args=-Dbazel.DigestFunction=sha256"
     # use remote caching for all the things
     echo "build --experimental_remote_spawn_cache"
-    # point it at our http cache ...
-    echo "build --remote_http_cache=${CACHE_HOST}"
     # don't fail if the cache is unavailable
     echo "build --remote_local_fallback"
-    # make sure the cache considers host toolchain versions
-    # NOTE: these assume nobody installs new host toolchains afterwards
-    # if $CC is set bazel will use this to detect c/c++ toolchains, otherwise gcc
-    # https://blog.bazel.build/2016/03/31/autoconfiguration.html
-    CC="${CC:-gcc}"
-    CC_VERSION=$(command_to_version $CC)
-    echo "build --action_env=CACHE_GCC_VERSION=${CC_VERSION}"
-    # NOTE: IIRC some rules call python internally, this can't hurt
-    PYTHON_VERSION=$(command_to_version python)
-    echo "build --action_env=CACHE_PYTHON_VERSION=${PYTHON_VERSION}"
+    # point it at our http cache ...
+    # NOTE our cache is versioned by the first path segment
+    # TODO(bentheelder): update the nursery deployment to something that supports this
```

Comment:
what does this mean?

Reply:
the cache server deployment "nursery" is actually https://github.com/buchgr/bazel-remote currently which only supports a single cache directory

Reply:
this is done now